replica/database.cc

```diff
@@ -1013,16 +1013,9 @@ void database::remove(const table& cf) noexcept {
     }
 }
 
-future<> database::drop_column_family(const sstring& ks_name, const sstring& cf_name, timestamp_func tsf, bool snapshot) {
-    auto uuid = find_uuid(ks_name, cf_name);
-    lw_shared_ptr<table> cfptr;
-    try {
-        cfptr = _column_families.at(uuid);
-        drop_repair_history_map_for_table(uuid);
-    } catch (std::out_of_range&) {
-        on_internal_error(dblog, fmt::format("drop_column_family {}.{}: UUID={} not found", ks_name, cf_name, uuid));
-    }
-    table& cf = *cfptr;
+future<> database::drop_column_family(table& cf, timestamp_func tsf, bool snapshot) {
+    auto uuid = cf.schema()->id();
+    drop_repair_history_map_for_table(uuid);
```

Comment:
This should be the last step, since it can race with a compaction completing just now.

Anyway, no need to touch this for this series.

Reply:
Makes sense